### PR TITLE
Adding logging to the website

### PR DESCRIPTION
**Pull request by jzhang-gp**
Use GCP cloud logging to emit logs to the Logging service. 

Only issue right now is that somehow the celery logs are not emitted

### CODE REVIEW

File: backend/__init__.py [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+
 from db.config import DevelopmentConfig
 from db.model import db
 from .auth import auth
 
+from google.cloud.logging.handlers import CloudLoggingHandler, setup_logging
+
+client = glog.Client()
+
+handler = CloudLoggingHandler(client, name="alchemy-backend")
```
Comment (edu-gp) [inline backend/__init__.py:18]: Can we extract the name to an env var?

File: docker/Dockerfile [hunk 2]
```diff
@@ -9,6 +9,15 @@ RUN pip install mockito flask-sqlalchemy alembic Flask-Testing psycopg2-binary
...
 RUN pip install google-cloud-secret-manager simple-salesforce PyJWT cryptography requests tldextract
 
+# GCP Cloud logging
+RUN pip install google-cloud-logging
+
+# AWS CLI
+RUN pip install boto3
+
+# Spring3 data fetcher
+RUN pip install ijson tqdm google-cloud-storage
```
Comment (edu-gp) [inline docker/Dockerfile:19]: Can we move the deps for AWS to another change? Doesn't look like they belong here.

File: frontend/__init__.py [hunk 3]
```diff
@@ -6,13 +6,23 @@
...
 from .auth import login_required
 
 from db._task import _Task
 from ar.data import fetch_tasks_for_user, fetch_tasks_for_user_from_db
 
+from google.cloud.logging.handlers import CloudLoggingHandler, setup_logging
+
+client = glog.Client()
+
+handler = CloudLoggingHandler(client, name="alchemy-frontend")
```
Comment (edu-gp) [inline frontend/__init__.py:22]: Ditto here, extract to env var

Comment (jzhang-gp) [reply frontend/__init__.py]: Done.

File: docker/Dockerfile (hunk 2 above)
Comment (jzhang-gp) [reply docker/Dockerfile:16]: Removed.

File: backend/__init__.py (hunk 1 above)
Comment (jzhang-gp) [reply backend/__init__.py]: Done.

**Pull request merged**